src/vs/workbench/browser/parts/editor/editorGroupsControl.ts

```diff
@@ -1564,6 +1564,7 @@ export class EditorGroupsControl extends Themable implements IEditorGroupsContro
 	private updateFromDropping(element: HTMLElement, isDropping: boolean): void {
 		const groupCount = this.stacks.groups.length;
 		const background = this.getColor(isDropping ? EDITOR_DRAG_AND_DROP_BACKGROUND : groupCount > 0 ? EDITOR_GROUP_BACKGROUND : null);
+		isDropping ? DOM.addClass(element, 'dropping') : DOM.removeClass(element, 'dropping');
```

Comment:
@Dari-K isn't this rather "`dragged-over`"?

Reply:
Hmm, not exactly. The element that the class will be applied to is the container for all editor panes/tabs. When you drag over the tab list the class is applied (but not to the tab list, to the container mentioned earlier), when you drag over any of the editor panes the class is removed. In either case you are still dragging over the container the class is applied to. If you drag over anything else the class just retains its previous state.